test/auditor/AuditorSpec.scala

```diff
@@ -19,12 +19,11 @@ package auditor
 import audit.Auditor
 import audit.returns.{AmendStarted, ReturnStarted}
 import base.utils.ConnectorISpec
-import org.mockito.ArgumentMatchers.any
+import org.mockito.ArgumentMatchers.{any, eq => eqTo}
```

Comment:
we may want to use the ArgumentMatcherSuger here which has already an eqTo member